Unable to delete pv and pvcs

After attempting to `kubectl delete -f example-cluster.yaml`, the operator logs indicate an exception as follows (deploying to GKE):

```time="2018-05-31T15:51:39Z" level=info msg="Found 0 existing clusters "
time="2018-05-31T15:51:39Z" level=info msg="Watching for elasticsearch events..."
time="2018-05-31T15:52:25Z" level=info msg="Process Elasticsearch Event ADDED"
time="2018-05-31T15:52:25Z" level=info msg="--------> Received ElasticSearch Event!"
time="2018-05-31T15:52:25Z" level=info msg="Found cluster: example-es-cluster"
time="2018-05-31T15:52:25Z" level=info msg="use-ssl not specified, defaulting to UseSSL=true"
time="2018-05-31T15:52:25Z" level=info msg="Using [upmcenterprises/docker-elasticsearch-kubernetes:6.1.3_1] as image for es cluster"
time="2018-05-31T15:52:25Z" level=info msg="use-ssl not specified, defaulting to UseSSL=true"
time="2018-05-31T15:52:25Z" level=info msg="Creating new certs!"
time="2018-05-31T15:52:25Z" level=info msg="Creating ca cert..."
time="2018-05-31T15:52:25Z" level=info msg="Creating node cert..."
time="2018-05-31T15:52:25Z" level=info msg="Creating kibana cert..."
time="2018-05-31T15:52:25Z" level=info msg="Creating cerebro cert..."
time="2018-05-31T15:52:26Z" level=info msg="Converting node to pkcs12..."
time="2018-05-31T15:52:26Z" level=info msg="Converting ca cert to jks..."
time="2018-05-31T15:52:26Z" level=info msg="Converting node cert to jks..."
time="2018-05-31T15:52:27Z" level=info msg="Discovery Service not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="Service es-data-svc-example-es-cluster not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="elasticsearch-example-es-cluster not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="cerebro-example-es-cluster not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="kibana-example-es-cluster not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="Deployment es-client-example-es-cluster not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="StatefulSet es-master-example-es-cluster-default not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="StatefulSet es-data-example-es-cluster-default not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="kibana-example-es-cluster not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="Deployment cerebro-example-es-cluster not found, creating..."
time="2018-05-31T15:52:27Z" level=info msg="--------> ElasticSearch Event finished!"
time="2018-05-31T16:21:26Z" level=info msg="Process Elasticsearch Event DELETED"
time="2018-05-31T16:21:26Z" level=info msg="--------> Deleting elasticSearch Cluster ...removing all components..."
time="2018-05-31T16:21:26Z" level=info msg="Scaled deployment: es-client-example-es-cluster to zero"
time="2018-05-31T16:21:26Z" level=info msg="Deleted deployment: es-client-example-es-cluster"
time="2018-05-31T16:21:26Z" level=info msg="Deleted replica set: es-client-example-es-cluster-5ddc8b9c89"
time="2018-05-31T16:21:26Z" level=info msg="Scaled deployment: kibana-example-es-cluster to zero"
time="2018-05-31T16:21:26Z" level=info msg="Deleted deployment: kibana-example-es-cluster"
time="2018-05-31T16:21:26Z" level=info msg="Deleted replica set: kibana-example-es-cluster-54b44bb9d9"
time="2018-05-31T16:21:26Z" level=info msg="Scaled deployment: cerebro-example-es-cluster to zero"
time="2018-05-31T16:21:26Z" level=info msg="Deleted deployment: cerebro-example-es-cluster"
time="2018-05-31T16:21:26Z" level=info msg="Deleted replica set: cerebro-example-es-cluster-5c99bf894"
time="2018-05-31T16:21:27Z" level=info msg="Scaled statefulset: es-master-example-es-cluster-default to zero"
time="2018-05-31T16:21:27Z" level=info msg="Deleted statefulset: es-master-example-es-cluster-default"
time="2018-05-31T16:21:27Z" level=info msg="Scaled statefulset: es-data-example-es-cluster-default to zero"
time="2018-05-31T16:21:27Z" level=info msg="Deleted statefulset: es-data-example-es-cluster-default"
time="2018-05-31T16:21:27Z" level=info msg="Deleted service: elasticsearch-discovery-example-es-cluster"
time="2018-05-31T16:21:27Z" level=info msg="Deleted service: es-data-svc-example-es-cluster"
time="2018-05-31T16:21:27Z" level=info msg="Deleted service: elasticsearch-example-es-cluster"
time="2018-05-31T16:21:27Z" level=info msg="Deleted service: cerebro-example-es-cluster"
time="2018-05-31T16:21:27Z" level=info msg="Deleted service: kibana-example-es-cluster"
time="2018-05-31T16:21:27Z" level=info msg="Deleted storageclasses"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xecd52b]

goroutine 42 [running]:
github.com/upmc-enterprises/elasticsearch-operator/pkg/processor.(*Processor).deleteElasticSearchCluster(0xc4202eba40, 0xc4201fa400)
	/home/steve/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/processor/processor.go:450 +0xaeb
github.com/upmc-enterprises/elasticsearch-operator/pkg/processor.(*Processor).processElasticSearchClusterEvent(0xc4202eba40, 0xc4201fa400, 0x0, 0x0)
	/home/steve/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/processor/processor.go:239 +0x1c1
github.com/upmc-enterprises/elasticsearch-operator/pkg/processor.(*Processor).WatchElasticSearchClusterEvents.func1(0xc4200820c0, 0xc4202eba40, 0xc42004e840, 0xc42006c600, 0xc42026e930)
	/home/steve/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/processor/processor.go:85 +0x223
created by github.com/upmc-enterprises/elasticsearch-operator/pkg/processor.(*Processor).WatchElasticSearchClusterEvents
	/home/steve/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/processor/processor.go:81 +0x78
rpc error: code = Unknown desc = Error: No such container: 2e204ee54dbdf85f2b2ac0b9e7999a3c26b8fd795e1f40120fc787d5aea86f22%```

--- a/pkg/processor/processor.go
+++ b/pkg/processor/processor.go
@@ -451,7 +451,7 @@ func (p *Processor) deleteElasticSearchCluster(c *myspec.ElasticsearchCluster) {
 
 	p.clusters[fmt.Sprintf("%s-%s", c.ObjectMeta.Name, c.ObjectMeta.Namespace)].Scheduler.Stop()
 
-	if !c.Spec.KeepSecretsOnDelete && *c.Spec.UseSSL {
+	if !c.Spec.KeepSecretsOnDelete {
 		if err := p.k8sclient.DeleteCertsSecret(c.ObjectMeta.Namespace, c.ObjectMeta.Name); err != nil {
 			logrus.Errorf("Could not delete cert secret %s: %v", c.ObjectMeta.Name, err)
 		}

